roles/ceph-nfs/tasks/pre_requisite_container.yml

```diff
@@ -37,3 +37,20 @@
   when:
     - ansible_os_family == 'RedHat'
     - sestatus.stdout != 'Disabled'
+
+- name: create dbus service file
+  become: true
+  template:
+    src: "{{ role_path }}/templates/org.ganesha.nfsd.conf"
+    dest: /etc/dbus-1/system.d/org.ganesha.nfsd.conf
+    owner: "root"
+    group: "root"
+    mode: "0644"
+  when:
+    - ceph_nfs_dynamic_exports
+
+- name: reload dbus configuration
+  shell:
```

Comment:
you should use the `command` module instead.

Reply:
done